pkg/builtin/types.go

```diff
@@ -45,4 +45,5 @@ const (
 	DayOfYear
 	Month
 	Weekday
+	Substring
```

Comment:
Put together with other string relevant ones

Reply:
OK, I'll modify it